### PR TITLE
chore: register containerd component using Component Governance

### PR DESCRIPTION
**Pull request by mainred**

### CODE REVIEW

File: vhdbuilder/cgmanifest.json [hunk 1]
```diff
@@ -10,6 +10,17 @@
                 }
             },
             "DevelopmentDependency": false
+        },
```
Comment (mainred) [inline vhdbuilder/cgmanifest.json:13]: The file name is "cgmanifest.json" and you can have as many as you need and can be anywhere in your repository.

File: NOTICE.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
-NOTICES
+# NOTICES
 
-This repository incorporates material as listed below or described in the code. 
+This repository incorporates material as listed below or described in the code.
 
 ## CalicoWindows
 
-**Source**: [https://github.com/projectcalico/calico](https://github.com/projectcalico/calico)
+**Source**: <https://github.com/projectcalico/calico>
```
Comment (mainred) [inline NOTICE.md:7]: They have the same result but with less code.

File: vhdbuilder/cgmanifest.json [hunk 3]
```diff
@@ -10,6 +10,17 @@
...
+        {
+            "Component": {
+                "Type": "other",
+                "Other": {
+                    "Name": "ContainerDWindows",
```
Comment (mainred) [inline vhdbuilder/cgmanifest.json:18]: In this repo, Linux is also using containerd, but we here just cover windows contained only

File: NOTICE.md [hunk 4]
```diff
@@ -209,4 +210,204 @@ This repository incorporates material as listed below or described in the code.
    See the License for the specific language governing permissions and
    limitations under the License.
 
-```
\ No newline at end of file
+```
+
+## ContainerDWindows
```
Comment (AbelHu) [inline NOTICE.md:215]: Normally d is lowercase

Comment (AbelHu) [reply NOTICE.md:215]: Same in cgmanifest.json

**Review by AbelHu — APPROVED** [review]

**Comment by mainred** [comment]
Could you please take a look at the manifest change for OSS registration, @microsoftopensource ? Thanks.

**Pull request merged**